frontend/apps/reader/modules/readerhighlight.lua

```diff
@@ -596,10 +611,13 @@ function ReaderHighlight:onPanelZoom(arg, ges)
     return true
 end
 
+local function isDocumentComicOrManga(file)
+    local filetype = util.getFileNameSuffix(file)
+    return filetype == "cbz" or filetype == "djvu"
```

Comment:
I can see treating CBZ as a comic by default, but DjVu is basically the same as PDF here? I read most of my manga/comics in PDF. On a related note, we also support CBT.

Reply:
`filetype:match("^cb[zrt7]") or filetype=="zip"`

Reply:
We don't currently support CBR and CB7(or do we? :D), and zip is potentially a bit of an issue for zipped FB2 or something?

But, yeah, +1 on the cb pattern regardless ;).

Reply:
@NiLuJe mupdf can do it, provided it's built that way. As for zip, it's extremely common format for ... "certain" kinds of manga :>

That said whether it's zip or fb2 inside is discerned by mupdf again, so perhaps more reliable method would be to inquire mupdf about what's it doing.

Reply:
I think for now, it's better not to support `djvu`, since they sometimes fail, when `cbz` works (due to lower resolution believe). I just tested `cbt` and it works without problems. I'm not sure about `cb7`, but `cbr` isn't supported, right?

Reply:
@NiLuJe zipped fb2 don't have extension of `.fb2.zip`?

Reply:
Perhaps a better solution would be to add a separate menu entry, where we let user decide which formats they want supported?

Reply:
I think this whole notion of support is weird when it seems like a feature similar to reflow. We talked about defaults earlier.

Reply:
> Perhaps a better solution would be to add a separate menu entry, where we let user decide which formats they want supported?

Strange, GH didn't show this to me. Anyway, I'd say it should be a per document thing (cf. reflow), possibly with a per format default.